app/controllers/chores_controller.rb

```diff
@@ -0,0 +1,37 @@
+class ChoresController < ApplicationController
+  def create
+    begin
+      chore = Chore.create(chore_params)
+      render status: 201,  json: chore.to_json
+    rescue
+      render status: 400
+    end
+  end
+
+  def destroy
+    begin
+      params.require(:id)
+      Chore.destroy_by(id: params[:id])
+      render status: 200
+    rescue
+      render status: 400
+    end
+  end
+
+  def update
+    begin
+      id = params[:id]  
```

Comment:
In some of these methods you're assuming that a chore / user / hideout exists. Please add comments in the methods where you assume this so that when we make the middleware we can adapt appropriately.